paycalculator/Program.cs

```diff
@@ -1,47 +1,141 @@
-﻿using Newtonsoft.Json;
-using System;
+﻿using Employees;
 using Employees.Entites;
+using Employees.Entities;
+using Employees.Entities.JSON;
 using Employees.Services;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
 
 namespace paycalculator
 {
     class Program
     {
-        
+        private static NewEmployeeInput _newEmployeeInput = new NewEmployeeInput();
+        private static ILog iLogs;
+        private static List<ILog> _Logs = new List<ILog>();
 
-        public static void Main(string[] args)
+        static void Main(string[] args)
         {
+            DisplayEmployees displayEmployees = new DisplayEmployees();
+            displayEmployees.DisplayContent(displayEmployees.permanentEmployee1, displayEmployees.PermanentEmployee2, displayEmployees.TempEmployee1);
+            FindLogFiles findLogs = new FindLogFiles();
+            iLogs = new EmployeeCreatedLog(findLogs.EmployeeCreatedLogsEndPath);
```

Comment:
Your use of the `iLogs` static field is a bit confusing here - why isn't it a local variable? why is it static? why is it set to multiple different things? You actually don't need it, you can just do `_Logs.Add(new EmployeeCreatedLog(findLogs.EmployeeCreatedLogsEndPath);`